docs/api/tutorials/Player Tips and Tricks.md

```diff
@@ -0,0 +1,35 @@
+The **Player Tips and Tricks** section serves as a valuable resource for users seeking guidance on optimizing their experience with Pillarbox.
+
+## Adding or Overriding Resource Properties
+
```

Comment:
Since this section is specific to the srgssr middleware I would add a notice like:

```suggestion

> This mechanism is specific to media content from SRG SSR

```